wordpress-mobile/WordPress-Android
# Updated stories library hash to latest

Somehow the stories library hash was downgraded back to the latest on November 27th instead of pointing to stories-library's `develop`.
- This is the [commit history](https://github.com/wordpress-mobile/WordPress-Android/commits/4b12e411e3cbed39a9b3bd8a79b09179f8dc4e08/libs) for the `16.3` tag. 
- And this is the https://github.com/wordpress-mobile/WordPress-Android/commits/release/16.4/libs for `release/16.4`
I can see the history diverges after November 27th, although it was merged back into develop in https://github.com/wordpress-mobile/WordPress-Android/pull/13590

Somehow some merge afterwards missed the right conflict resolution but, given there were no compilation issues it all just worked out.

To test:
 - make sure this all works https://github.com/wordpress-mobile/WordPress-Android/pull/13539 (I re-reviewed it and made sure already)
 - a quick way to know is to make sure the upload icon  in the camera capture mode does appear when adding an empty Story block to a Gutenberg post and tapping "Add Media" as shown here https://github.com/wordpress-mobile/WordPress-Android/pull/13543

PR submission checklist:
